pkg/controllers/management/aks/aks_cluster_handler.go

```diff
@@ -67,9 +73,36 @@ func Register(ctx context.Context, wContext *wrangler.Context, mgmtCtx *config.M
 		DynamicClient:        aksCCDynamicClient,
 		ClientDialer:         mgmtCtx.Dialer,
 		Discovery:            wContext.K8s.Discovery(),
-	}}
+	},
+		wContext.Core.Secret(),
```

Comment:
Write out the field name for the struct now that it isn't just one embedded field, will be easier to read.

Reply:
done